design/model_design.md

```diff
@@ -0,0 +1,60 @@
+## Model Schema
+
+This schema is designed to allow for power and flexibility in the API given its purpose may either change or expand in the near future.
+
+### TranslationService
+
+This is primary interface model in the system, describing API methods, which each translation service will implement. 
+
+**Fields**
+* API_KEY (string)
+* BASE_URL (string)
+* price per 1000 symbols (integer)

```

Comment:
It may be variable value. And mistakes can be occurred, when we will want to calculate the price of the past translations. Moreover, some services may provide the trial or discount translation for the first `n` symbols each months/week/etc